gcp/live/dev/k8s/gpii/couchdb-prometheus-exporter/terraform.tfvars

```diff
@@ -0,0 +1,19 @@
+# ↓ Module metadata
+terragrunt = {
+  terraform {
+    source = "/project/modules//couchdb-prometheus-exporter"
+  }
+
+  dependencies {
+    paths = [
+      "../couchdb",
+    ]
+  }
+
+  include = {
+    path = "${find_in_parent_folders()}"
+  }
+}
+
+# ↓ Module configuration (empty means all default)
+replica_count = 1
```

Comment:
This is already a default value, why repeat?

Reply:
Reading further, I noticed it's not a default in the `gcp/modules/couchdb-prometheus-exporter` module, maybe make it default there as well to avoid repetition in envs?

Reply:
Done.